assembly/src/parsers/io_ops.rs

```diff
@@ -287,6 +271,7 @@ fn push_mem_addr(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Assemb
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::parsers::{BaseElement, FieldElement};
```

Comment:
nit: I think these can come from just `crate` (no need to go to `parsers` as well).

Reply:
cargo build failed if I don't include parsers actually?

Reply:
cargo test I mean

Reply:
oh - i see. It's fine as is then.